contracts/test/LibPerpetualsMock.sol

```diff
@@ -0,0 +1,14 @@
+pragma solidity ^0.8.0;
+
+import "../lib/LibPerpetuals.sol";
+
+contract LibPerpetualsMock {
+    function canMatch(
+        Perpetuals.Order calldata a,
+        uint256 aFilled,
+        Perpetuals.Order calldata b,
+        uint256 bFilled
+    ) public view returns (bool) {
```

Comment:
Seeking clarification here on whether these should be declared `external` or `public`?

Reply:
Would say it doesn't matter since they are just used for testing

Reply:
@sporejack probably should be external, I'll change it just for consistency. But as Ben mentioned, this is just for testing so its not too critical.